README.md

```diff
@@ -5,6 +5,10 @@
 </p>
 <br/>
 
+## Overview
+
+The Babylon Wallet Connector provides a unified interface for integrating both Bitcoin (BTC) and Babylon (BBN) wallets into Babylon dApp. It supports both native wallet extensions and injectable mobile wallets.
```

Comment:
Lines should be wrapped at 80 characters on .md files

Reply:
Okay, will do

Reply:
Done